Example/IGListKitExamples/ViewControllers/SingleSectionViewController.swift

```diff
@@ -76,7 +76,7 @@ final class SingleSectionViewController: UIViewController, IGListAdapterDataSour
     // MARK: - IGListSingleSectionControllerDelegate
     
     func didSelect(_ sectionController: IGListSingleSectionController) {
-        let section = adapter.section(for: sectionController) + 1
+        let section = adapter.section(for: sectionController) + UInt(1)
```

Comment:
Are you working off of an old revision? It returns an `NSInteger`

https://github.com/Instagram/IGListKit/blob/master/Source/IGListAdapter.h#L157